lib/teleterm/api/protogen/js/v1/service_grpc_pb.d.ts

```diff
@@ -6,14 +6,13 @@
 
 import * as grpc from "grpc";
 import * as v1_service_pb from "../v1/service_pb";
-import * as google_protobuf_empty_pb from "google-protobuf/google/protobuf/empty_pb";
+import * as v1_app_pb from "../v1/app_pb";
+import * as v1_auth_settings_pb from "../v1/auth_settings_pb";
```

Comment:
The updated JS files have to be manually updated in the webapps repo (for now at least). I made a PR (gravitational/webapps#1155) which does exactly that.

Though looking at the changes in those JS files, nothing should break if we merge this PR without merging the one that I created. It looks like it's just a bunch of reordered statements.